java/vector/src/test/java/org/apache/arrow/vector/ITTestLargeVector.java

```diff
@@ -45,32 +47,33 @@ private static void testLargeLongVector() {
         BigIntVector largeVec = new BigIntVector("vec", allocator)) {
       largeVec.allocateNew(vecLength);
 
-      System.out.println("Successfully allocated a vector with capacity " + vecLength);
+      logger.info("Successfully allocated a vector with capacity {}", vecLength);
 
       for (int i = 0; i < vecLength; i++) {
         largeVec.set(i, i * 10L);
 
         if ((i + 1) % 10000 == 0) {
-          System.out.println("Successfully written " + (i + 1) + " values");
+          logger.debug("Successfully written {} values", i + 1);
         }
       }
-      System.out.println("Successfully written " + vecLength + " values");
+      logger.info("Successfully written {} values", vecLength);
 
       for (int i = 0; i < vecLength; i++) {
         long val = largeVec.get(i);
         assertEquals(i * 10L, val);
 
         if ((i + 1) % 10000 == 0) {
-          System.out.println("Successfully read " + (i + 1) + " values");
+          logger.debug("Successfully read {} values", i + 1);
         }
       }
-      System.out.println("Successfully read " + vecLength + " values");
+      logger.info("Successfully read {} values", vecLength);
```

Comment:
I suggest you downgrade all these to trace (and those above). They shouldn't really present on normal test runs.

Reply:
done